README.md

```diff
@@ -1,101 +1,171 @@
 # node-test-runner [![Version](https://img.shields.io/npm/v/elm-test.svg)](https://www.npmjs.com/package/elm-test)
 
-Runs [elm-test](https://package.elm-lang.org/packages/elm-explorations/test/latest) suites from Node.js
+Runs [elm-explorations/test] suites from Node.js.
+
+When people say “elm-test” they usually refer to either:
+
+- This CLI tool for running tests.
+- [elm-explorations/test] – an Elm package for defining tests that this CLI tool can run.
+
+[elm-explorations/test]: https://package.elm-lang.org/packages/elm-explorations/test/latest
 
 ## Installation
 
-```shell
-npm install -g elm-test
+```
+npm install --save-dev elm-test
```

Comment:
Yes, it’s about having your project just work down the road:

- When you return to your project a year later you want it to just work, not debug why your tests suddenly won’t run (that should be done as a separate “update elm-test” task).
- The project should just work for new contributors – no “works on my machine” issues!

The things that can go wrong over time are:

- A new Elm version, like you say. You have to use an elm-test version that is compatible with your Elm version. So people should install Elm locally as well (I’m biased of course, being the author of [elm-tooling](github.com/elm-tooling/elm-tooling-cli/)).
- Accidentally relying on a bug in elm-test that gets fixed in a later version, like in https://discourse.elm-lang.org/t/do-you-put-tests-outside-the-tests-folder/6386/20?u=lydell
- elm-test introducing a regression in a new version that is a deal-breaker in your project

But since Evan seems to mostly push for using the Elm installers, maybe we should add a note for how to install globally as well?

Reply:
Okay I am almost sold. One last question: what if a user is writing only elm (no js) and doesn't want to add three extra entries (package.json, lock file and node_modules) to their top level directory?

Reply:
I guess this here is only advice, it is pretty obvious what someone should do to avoid those entries (global install)

Reply:
Great questions!

So we used to only mention `npm install -g elm-test` (global install), but people who know their stuff still installed locally (like me, and the one who set up elm-test at work, and many Elm package repos). So it does sound like people who know what they’re doing and have an opinion are able to take instructions as advice, like you say, and think for themselves!

So it might be worth optimizing for people who don’t know the ins and outs of npm yet. I think were pushing them better towards the pit of success if the copy-pasteable command is local? And presenting both a local and a global install command just make the reader make a choice they don’t want to do?

Reply:
yep, sounds good.

No choice is better than a choice when we want to get people started as smoothly as possible 👍 and I am happy that it is clear enough to people who care about not having package.json files in their repo what they need to do.